### DISCUSSION TITLE
[Feature] Add storage support and E2E test for InfluxDB 2.x

### DISCUSSION THREAD

**Discussion by linghengqian** [apache/skywalking#8891] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I hope SkyWalking OAP Server will add storage support and E2E testing for InfluxDB 2.x.

### Use case

- I hope SkyWalking OAP Server will add storage support and E2E testing for `InfluxDB 2.x` . The current https://github.com/apache/skywalking/tree/master/test/e2e-v2/cases/storage/influxdb only has the E2E test of `InfluxDB 1.x`. 

- At the same time, there are still some issues in the community indicating that the current `SkyWalking OAP Server` version  backend storage does not support `InfluxDB 2.x`, refer to https://github.com/apache/skywalking/issues/8163 . I hope `SkyWalking OAP Server` can add storage support for `InfluxDB 2.x` and E2E testing. 

- The corresponding image is at https://hub.docker.com/_/influxdb .

### Related issues

Yes, refer to https://github.com/apache/skywalking/issues/8163 .

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I think before you submit anything, could you make a clear what is going to do? The First would be, is InfluxDB 2.x client compatible with InfluxDB 1.x? All the libraries are going to exist in one JVM and the same class loader, if they conflict with the official ElasticSearch's client, then you face a serious problem.

cc @dmsolr PMC member leading on InfluxDB 1.x support.

**Comment by linghengqian** [comment]
> I think before you submit anything, could you make a clear what is going to do? The First would be, is InfluxDB 2.x client compatible with InfluxDB 1.x? All the libraries are going to exist in one JVM and the same class loader, if they conflict with the official ElasticSearch's client, then you face a serious problem.
> 
> cc @dmsolr PMC member leading on InfluxDB 1.x support.

I'm pretty sure the InfluxDB 2.x client is not compatible with the InfluxDB 1.x client. But since I'm not familiar with the internals of the InfluxDB 2.x client, I can't submit the relevant PR. Refer to https://github.com/influxdata/influxdb-client-java .

**Comment by wu-sheng** [comment]
If you are not going to submit anything, #8163 had given a very clear answer. No, we don't support that.

`Feature` labeled issue is for contributors to track the progress. 
It looks like this is only a discussion, nothing should be tracked here.

I am moving to the discussion panel.

**Comment by linghengqian** [comment]
Just a reminder of the update, I didn't continue any work in this discussion due to the removal of support in the https://github.com/apache/skywalking/pull/9060 .
